src/concepts/components/internalstate.md

```diff
@@ -0,0 +1,169 @@
+---
+description: Component can maintain it's own state and render information depending on it
+---
+
+# Internal State
```

Comment:
The capitalisation of this title isn't consistent with some of the other titles.